src/components/Task/TaskRow/TaskRow.tsx

```diff
@@ -290,28 +301,50 @@ export const TaskRow: React.FC<TaskRowProps> = ({
           </Hidden>
           <Hidden mdDown>
             <TaskDate isComplete={isComplete} taskDate={taskDate} />
-            <Box onClick={(e) => e.stopPropagation()}>
-              <TaskCommentsButton
-                isComplete={isComplete}
-                numberOfComments={comments?.totalCount}
-                onClick={handleCommentButtonPressed}
-                onMouseEnter={() => preloadTaskModal(TaskModalEnum.Comments)}
-              />
-            </Box>
-          </Hidden>
-
-          <Hidden mdUp>
-            <Box>
-              <TaskDate isComplete={isComplete} taskDate={taskDate} small />
-              <Box>
+            <Tooltip
+              title={
+                comments.totalCount ? (
+                  <CommentTooltipText comments={comments.nodes} />
+                ) : null
+              }
+              placement="top"
+              arrow
+            >
+              <Box onClick={(e) => e.stopPropagation()}>
                 <TaskCommentsButton
                   isComplete={isComplete}
                   numberOfComments={comments?.totalCount}
                   onClick={handleCommentButtonPressed}
                   onMouseEnter={() => preloadTaskModal(TaskModalEnum.Comments)}
-                  small
                 />
               </Box>
+            </Tooltip>
+          </Hidden>
+
+          <Hidden mdUp>
```

Comment:
Is this the same as the above, just with the `small` prop on `TaskCommentsButton`? If so, maybe we could do a media query to determine whether `small` should be true or false instead of having to duplicate all the components. I didn't look super carefully, so there might be more differences I missed.

Reply:
Great point! I didn't notice how similar they were.